### PR TITLE
bluetooth: tester: Update btp for codec and qos config

### PR DESCRIPTION
**Pull request by mkasenberg**
Update BTP_ASCS_CONFIGURE_CODEC to send codec specific configuration in LTV format.

Update parameters of BTP_ASCS_CONFIGURE_QOS to match the BT spec defined lengths.

Restore nrf5340_hci_rpmsg.conf. I moved recently its content into overlay-le-audio.conf, but forgot that it is still useful for non LE Audio tests.

### CODE REVIEW

File: tests/bluetooth/tester/nrf5340_hci_rpmsg.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Those have to be the same as in the controller (hci_rpmsg)
```
Comment (Thalley) [inline tests/bluetooth/tester/nrf5340_hci_rpmsg.conf:1]: Any way we can automatically keep these in sync? 

Also, the `hci_rpmsg` has quite a few different config files for the nRF5340; which one of them are we syncing this file with?

File: tests/bluetooth/tester/overlay-le-audio.conf [hunk 2]
```diff
@@ -22,7 +22,7 @@ CONFIG_BT_ASCS_ASE_SNK_COUNT=2
 CONFIG_BT_ASCS_ASE_SRC_COUNT=2
 
 # Support an ISO channel per ASE
-CONFIG_BT_ISO_MAX_CHAN=4
+CONFIG_BT_ISO_MAX_CHAN=2
```
Comment (Thalley) [inline tests/bluetooth/tester/overlay-le-audio.conf:25]: Why the reduction here? Controller support?

**Review by Thalley — APPROVED** [review]

File: tests/bluetooth/tester/nrf5340_hci_rpmsg.conf (hunk 1 above)
Comment (mkasenberg) [reply tests/bluetooth/tester/nrf5340_hci_rpmsg.conf:1]: Maybe, somehow, do you have any idea?

The hci_rpmsg is built with nrf5340_cpunet_iso-bt_ll_sw_split.conf + CONFIG_BT_CTLR_CONN_ISO_LOW_LATENCY_POLICY=y

Comment (Thalley) [reply tests/bluetooth/tester/nrf5340_hci_rpmsg.conf:1]: One way we could do it, is to use the `nrf5340_cpunet_iso-bt_ll_sw_split.conf` as the main config file, and then add whatever we need on top of that as an overlay? 

I don't have any really good suggestions though.

**Review by sjanc — APPROVED** [review]
I agree that eventually we should have single default config for controller that would suit all tests, but for now I'm fine with overlay

**Review by carlescufi — APPROVED** [review]

**Pull request merged**